source/clear-linux/guides/tooling/mixer.rst

```diff
@@ -339,12 +330,47 @@ set to get a smaller kernel image, which will also be faster to load.
    Look in ~/mixer/update/www/<mix version> to see the update content in your
    workspace.
 
-Example 3: Deploy updates to target
+Example 3: Edit bundle content
```

Comment:
I think "Example 3" is good to have, but it's in an odd location.  Maybe, it should be placed after "Deploy Updates to Target".